src/app/to-do/to-do.component.html

```diff
@@ -8,34 +8,36 @@ <h1>Check It Off</h1>
 <h2>What do you need to get done today?</h2>
 
 <div class="todo">
-	<div class="list">
-		<input [(ngModel)]="newTask" type="text"
-		value="" class="taskBox">
-		<button (click)="addToList()" class="task-btn">
-			Add Task
-		</button>
-	</div>
+	<form [formGroup]="todoList" (ngSubmit)="onSubmit()" class="list">
+    <input type="text" id="task" name="task" formControlName="task" placeholder="task" class="taskBox">
+    <br>
+    <input type="text" id="timeEstimation" name="timeEstimation" formControlName="timeEstimation" placeholder="time estimate" class="timeBox" >
+    <label for="priority" class="priorityLabel">Priority:</label>
+    <select name="priority" id="priority" formControlName="priority" class="priority">
+      <option value="high">High</option>
+      <option value="low">Low</option>
+    </select>
+    <br>
```

Comment:
Avoid the use of `<br>` and other style only elements like `<strong>`, `<b>`, `<hr>`, etc. You should use CSS instead of these elements to move elements around on the page.